Grid-Stat/plt_gridstat_multidate_heatplot.py

```diff
@@ -363,7 +308,7 @@
 
 # save figure and display
 plt.savefig(OUT_PATH)
-plt.show()
+#plt.show()
```

Comment:
If we don't have live plotting, I might make some changes here where there will be options in terms of using the singularity environment or not.  As I mention, this is a breaking issue for me but I can resolve it when I merge with develop.

Reply:
I understand the issue with the live plotting. Is there anything you'd like me to change for this, or is this for further work beyond the internship?

Reply:
I think this will be ongoing work, just a note about this issue is fine.  Likely, the solution will depend on what we encounter moving this to Expanse.